docs/source/tools/codegen.rst

```diff
@@ -79,29 +84,22 @@ The above settings can be configured in the ``codegen`` element of the DAML proj
         output-directory: scala-codegen/src/main/scala
         verbosity: 2
 
-You can run the above configuration to generate **Java** code::
-
-    $ daml codegen java
-
-and to generate **Scala** code::
-
-    $ daml codegen scala
+You can then run the above configuration to generate your **Java** or **Scala** code::
 
-The equivalent **DAML to Java** command line configuration::
+    $ daml codegen [java|scala]
 
-    $ daml codegen java ./.daml/dist/quickstart-0.0.1.dar=com.daml.quickstart.iou --output-directory=java-codegen/src/main/java --verbosity=2
+The equivalent **Java** or **Scala** command line configuration would be::
 
-and **DAML to Scala** command line configuration::
+    $ daml codegen [java|scala| ./.daml/dist/quickstart-0.0.1.dar=com.daml.quickstart.iou --output-directory=java-codegen/src/main/java --verbosity=2
 
-    $ daml codegen scala ./.daml/dist/quickstart-0.0.1.dar=com.daml.quickstart.iou --output-directory=scala-codegen/src/main/scala --verbosity=2
+In order to compile the resulting **Java** or **Scala** classes, you need to
+add the corresponding dependencies to your build tools. 
 
-In order to compile the resulting **Scala** or **Java** classes, you need to
-add the corresponding dependencies to your build tools. For **Scala**, you can
-depend on::
+For **Scala**, you can depend on::
 
     "com.daml" %% "bindings-scala" % 1.0.1
```

Comment:
Either update it to 1.5.0 and add a comment that this should be updated to whatever SDK version you are using or use a placeholder and tell users to replace it.